tests/neg-custom-args/fatal-warnings/i9408a.scala

```diff
@@ -0,0 +1,73 @@
+import language.`3.0-migration`
+import scala.language.implicitConversions
+
+object Test1 {
+  implicit def implicitLength(s: String): Int = s.length
+  val length: Int = "qwerty" // ok
+}
+
+object Test2 {
+  implicit val implicitLength: Conversion[String, Int] = _.length
+  val length: Int = "qwerty" // ok
+}
+
+object Test3 {
+  implicit val implicitLength: String => Int = _.length
+  val length: Int = "qwerty" // error
+}
+
+object Test4 {
+  implicit def implicitLength: String => Int = _.length
+  val length: Int = "qwerty" // error
+}
+
+object Test5 {
+  implicit def implicitLength[A]: String => Int = _.length
+  val length: Int = "qwerty" // error
+}
+
+object Test6 {
+  implicit val implicitLength: Map[String, Int] = Map("qwerty" -> 6)
+  val length: Int = "qwerty" // error
+}
+
+object Test7 {
+  implicit def a2int[A](a: A)(implicit ev: A => Int): Int = a // error
+}
+
+object Test8 {
+  implicit def a2int[A](a: A)(implicit ev: A => Int): Int = ev(a) // ok
+}
+
+object Test9 {
+  implicit def a2int[A](a: A)(implicit ev: A <:< Int): Int = a // ok
+}
+
+object Test10 {
+  trait Foo {
+    def foo = "foo"
+  }
+  implicit def a2foo[A](a: A): Foo = new Foo {}
+  123.foo // ok
+}
+
+object Test11 {
+  trait Foo {
+    def foo = "foo"
+  }
+  implicit def a2foo[A]: A => Foo = _ => new Foo {}
+  123.foo // error
+}
+
+object Test12 {
+  implicit class FooOps(a: Any) {
+    def foo = "foo"
+  }
+  123.foo // ok
+}
+
+object Test13 {
+  def foo()(implicit x: String => Int) = ???
+  implicit val f: String => Int = _.size
+  foo()  // ok
```

Comment:
Here `f` is used as a normal implicit argument, therefore no warnings are expected.